.github/release-drafter-config.yml

```diff
@@ -0,0 +1,23 @@
+# Configuration for Release Drafter: https://github.com/toolmantim/release-drafter
+template: |
+  <!-- Optional: add a release summary here -->
+  $CHANGES
+
+name-template: $NEXT_MINOR_VERSION
+tag-template: $NEXT_MINOR_VERSION
+# Uses a more common 2-digit versioning in Jenkins plugins. Can be replaced by semver: $MAJOR.$MINOR.$PATCH
+version-template: $MAJOR.$MINOR
```

Comment:
We don't follow this versioning style. We have 4 parts to semantic versioning. 
Also remove any unnecessary comments

Reply:
Done.